internal/terminal/handler/keymap.go

```diff
@@ -0,0 +1,23 @@
+package handler
+
+import (
+	teakey "github.com/charmbracelet/bubbles/key"
+)
+
+type keyMap struct {
+	SwitchTabRight teakey.Binding
+	SwitchTabLeft  teakey.Binding
+	Quit           teakey.Binding
+}
+
+var keys = keyMap{
+	SwitchTabRight: teakey.NewBinding(
+		teakey.WithKeys("shift+right"),
+	),
+	SwitchTabLeft: teakey.NewBinding(
+		teakey.WithKeys("shift+left"),
+	),
+	Quit: teakey.NewBinding(
+		teakey.WithKeys("ctrl+q"),
+	),
```

Comment:
We use `ctrl+c`  to exit any program. This is breaking the generic action. Can you replace with `ctrl+c`?

Reply:
for sure 👍